addon/components/empty-wrapper.js

```diff
@@ -0,0 +1,8 @@
+import Component from '@ember/component';
+import layout from '../templates/components/empty-wrapper';
+
+export default Component.extend({
+  tagName: '',
+
+  layout,
```

Comment:
Curious as to what you all think of this, it's a workaround for creating templates like this:

```tsx
const labelledDropzoneMarkup = label ? (
  <Labelled
    id={id}
    label={label}
    action={labelAction}
    labelHidden={labelHidden}
  >
    {dropZoneMarkup}
  </Labelled>
) : (
  dropZoneMarkup
);
```

So we could use the `with` helper in templates like so:

```hbs
{{#with (if label
  (component "polaris-labelled"
    id=id
    label=label
    labelHidden=labelHidden
    action=labelAction
  )
  (component "empty-wrapper")
) as |wrapper|}}
  {{#component wrapper}}
    {{!-- dropzone markup here --}}
  {{/component}}
{{/with}}
```

without having to duplicate template logic, etc.

Reply:
We've built a `no-op` component in another addon which basically serves the same purpose, so perhaps we could look at moving that here?

Alternatively (and possibly better), I don't see any reason why using our existing `wrapper-element` component with `tagName=""` wouldn't work, could you give it a try?

Reply:
Ah I saw `wrapper-element` but didn't think to try passing a blank tag name. I'll see if that works 👍